src/section-solving-one-step-equations-and-inequalities.mbx

```diff
@@ -159,140 +159,209 @@
 	        <p>We can see from the above scale that <m>x=3</m> is obviously correct. Removing half of the weight on both sides of the scale is like dividing both sides of an equation by <m>2</m>:
 	        	<md>
 		        	<mrow>2x\amp=6</mrow>
-		        	<mrow>\frac{2x}{2}\amp=\frac{6}{2}</mrow>
+		        	<mrow>\frac{2x}{\highlight{{}2}}\amp=\frac{6}{\highlight{{}2}}</mrow>
```

Comment:
There is no need for `\highlight{{}2}`. It would just be `\highlight{2}`. With something like `\highlight{{}+2}`, the extra brace pair is there because the `+` is a binary operator, and should have something on either side. Otherwise it will be interpreted as a "positive sign", and the spacing between the `+` and things to each side of it will be off. So the extra brace pair is serving as an (empty) left operand for the `+` sign.